docs/examples/examples_06_triangle.md

```diff
@@ -11,18 +11,41 @@ kernelspec:
   name: python3
 ---
 
-(examples-facet)=
+(examples-triangle)=
 
-# Complex shapes from meshes
+# Complex shapes - The Triangle class
 
-## Application of the Facet class
+The field of a homogeneously charged magnet is, on the outside, the same as the field of a similarly shaped body with a magnetic surface charge. The surface charge is proportional to the projection of the magnetization vector onto the surface normal. The `Triangle` class is set up so that it can easily be used to approximate surfaces, and given the magnetization vector, the charge density is automatically computed. The resulting H-field is correct, but the B-field is only correct on the outside of the body, because on the inside the magnetization must be added to it.
 
-When a `Facet` object has a closed surface, where all individual facets are oriented outwards, their combined field corresponds to the one of a homogeneously charged magnet. Therefore, the `Facet` class can be used to approximate any form using a mesh of triangular facets. The following example shows an Icosahedron constructed from multiple facets. However, it must be understood that with such Facet objects an inside-outside check is not possible for observer points, so that the inside B-field is missing the magnetization.
+## Example: Triangular Prisma
+
+Consider a Prisma with triangular base that is magnetized orthogonal to the base. All surface normals of the sides of the prisma are orthogonal to the magnetization vector. As a result the sides do not contribute to the magnetic field because their charge desity dissappears. Only top and bottom surfaces contribute. One must be very careful when defining those surfaces in such a way that the surface normals point outwards. The following examples shows how the `Triangle` class can be used to create such a prisma.
+
+```{code-cell} ipython3
+import magpylib as magpy
+
+top = magpy.misc.Triangle(
+    magnetization=(0,0,1000),
+    vertices= ((-1,-1,1), (1,-1,1), (0,2,1)),
+)
+bott = magpy.misc.Triangle(
+    magnetization=(0,0,1000),
+    vertices= ((-1,-1,-1), (0,2,-1), (1,-1,-1)),
+)
+
+prisma = magpy.Collection(top, bott)
+
+prisma.show()
```

Comment:
Arrows are overlapping, let us use a wider gap between triangles or decrease the magnetization arrow size

Reply:
done